packages/aws-cdk/lib/api/aws-auth/account-cache.ts

```diff
@@ -83,6 +83,9 @@ export class AccountAccessKeyCache {
       // File doesn't exist or is not readable. This is a cache,
       // pretend we successfully loaded an empty map.
       if (e.code === 'ENOENT' || e.code === 'EACCES') { return {}; }
+      // File is not JSON, could be corrupted because of concurrent writes.
+      // Again, an empty cache is fine.
+      if (e instanceof SyntaxError) { return {}; }
```

Comment:
Im actually not sure this is because of the parallelism of the integ tests, i remember @rix0rrr and I encountered this issue a few month ago and ended up in a rabbit hole of sdk code. 

What if we set the `CDK_HOME` env variable to be unique per execution before we resort to this? I'm not strictly opposed to it but i feel like we are doing it for the wrong reasons now and we might be missing something else.

Reply:
This specifically has nothing to do with the SDK. This file is completely under our control, written and read by us only.

I can't imagine it being anything other than the parallelism we introduced recently, given that I haven't seen this error crop up before that.

I'm open to another explanation of the available facts if you have them, and I'll agree it breaks my mental model of how I thought NodeJS worked as well. Even concurrency isn't the ultimate root cause, I'm not concerned about this fix too much, which is why I'm hesitant to do more complicated workarounds.

This is all to say: I say we ship it :).

Reply:
Uh umm. It might also be this:

https://github.com/aws/aws-cdk/blob/ddff369bdf9a86730b724308cf84a0bde7417c46/packages/aws-cdk/lib/api/aws-auth/account-cache.ts#L92-L93

But even given that I don't mind too much to put this workaround in.

Reply:
Ship and may the force be with you 👍